doc/api/util.md

```diff
@@ -325,6 +325,60 @@ The `--throw-deprecation` command-line flag and `process.throwDeprecation`
 property take precedence over `--trace-deprecation` and
 `process.traceDeprecation`.
 
+## `util.diff(actual, expected)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `actual` {any}
+* `expected` {any}
+* Returns: {string | null} A string or `null` if there is difference
+  between `actual` and `expected`
```

Comment:
This is worded a bit oddly. As I understand it, a string describing the differences is returned if a difference is dectected, and `null` is returned if no difference is detected. Correct?

Reply:
@jasnell yeah you are right, no idea why it made sense in my head while writing it 😄